docs/developer-docs/daos/nns/using-the-nns-dapp/nns-dapp-importing-tokens.mdx

```diff
@@ -0,0 +1,67 @@
+
+## How to find canister IDs
+
+There are several ways to find ledger canister IDs of ICRC-1 tokens. The tutorial video above mentions two. This is a more comprehensive list.
+
+- [ICPSwap](https://info.icpswap.com/swap): Curated list of tokens traded on ICPSwap updated by the ICPSwap SNS DAO.
+- [ICP Tokens](https://icptokens.net/): Curated list of tokens traded on ICP-based DEXs updated by ICP Tokens.
+- [ICP Dashboard](https://dashboard.internetcomputer.org/): List of all chain-key tokens (under Chain Fusion menu option) updated automatically.
+- [nftGeek](https://t5t44-naaaa-aaaah-qcutq-cai.raw.icp0.io/tokens): Curated list of tokens traded on ICP-based DEXs updated by nftGeek.
+- [ICPEx](https://icpex.org/dashboard): Curated list of tokens traded on ICPEx updated by ICPEx.
+- [CoinMarketCap](https://coinmarketcap.com/): The ICRC tokens that are listed on CoinMarketCap provide their ledger canister ID under Contract.
+- [CoinGecko](https://coingecko.com/): The ICRC tokens that are listed on CoinGecko provide their ledger canister ID under Contract.
+
+
+## How to import ICRC tokens
+
+First, find the ledger canister ID of the token you want to import to your NNS dapp wallet. The list above covers most tokens deployed on the Internet Computer. Once you have its ledger canister ID, you may optionally look for its index canister to display the transaction history.
+
+:::info
+Importing an index canister is completely optional. Many popular tokens **do not have index canisters**.
+:::
+
+- **Step 1:**  Sign into the NNS dapp, and navigate to **Tokens**. At the bottom of the table, click **Import Token**.
+
+![NNS Import Tokens 1](../../_attachments/nns-import-tokens-1.webp)
+
+:::info
+If you don't want to see tokens you don't hold, you can click the settings button at the top right corner of the table, and enable **Hide zero balances**.
+:::
+
+- **Step 2:** Paste the ledger and index canister IDs to the corresponding input fields. If you don't provide the index canister ID now, you can add it any time in the future.
+
+![NNS Import Tokens 2](../../_attachments/nns-import-tokens-2.webp)
+
+- **Step 3**: Confirm that this is the token you want to import. By clicking the rightmost button, you can see the canister on the ICP dashboard.
+
+![NNS Import Tokens 3](../../_attachments/nns-import-tokens-3.webp)
+
+Once your token was successfully imported, you can send tokens to your principal ID.
+
+![NNS Import Tokens 4](../../_attachments/nns-import-tokens-4.webp)
+
+Going back, you should see all imported tokens in the tokens table.
+
+![NNS Import Tokens 5](../../_attachments/nns-import-tokens-5.webp)
+
+You can add and remove tokens any time. **Note**: removing a token doesn't remove the asset itself. If you remove a token that has a balance above 0, you can always re-import it, and the asset will be accessible.
```

Comment:
Is this maybe still something we want to recommend not doing?
Or at least say something like "you should remember which tokens you have" as the NNS dapp will not remember this if you just remove it or somesuch?